code/app/src/main/java/com/cmput301w21t06/crowdfly/Views/AddExperimentActivity.java

```diff
@@ -85,10 +85,10 @@ public void afterTextChanged(Editable editable) {
             @Override
             public void onClick(View view) {
                 //hard coding variables for now will be adjusted
-                String description = "binomial";
+                String description = getDescription();
                 String region = getRegion();
                 int minNumTrials = getMinNumTrials();
-                Experiment expAdd = new Experiment(description, region, minNumTrials,userID);
+                Experiment expAdd = new Experiment(description, region, minNumTrials,userID,"binomial");

```

Comment:
Lets use the string resource to store these strings.
"binomial", "measurement", "count"

Reply:
This will be fixed when this code is entirely refactored